ovs/lib/vdisk.py

```diff
@@ -1533,10 +1534,28 @@ def sync_with_reality(vpool_guid=None):
                             new_vdisk.pagecache_ratio = 1.0
                             new_vdisk.metadata = {'lba_size': new_vdisk.info['lba_size'],
                                                   'cluster_multiplier': new_vdisk.info['cluster_multiplier']}
+                            VDiskController._logger.info('vDisk {0} - has parent func {1}'.format(new_vdisk.name, hasattr(entry, 'parent')))
+                            if hasattr(entry, 'parent'):   # Older releases do not have this option
+                                # Parent is a volume ID
+                                parent_id = entry.parent()  # type: str
+                                VDiskController._logger.info('vDisk {0} - has parent with vol id {1}'.format(new_vdisk.name, parent_id))
+                                if parent_id:
+                                    # @todo figure out why this is empty for clone of clone
```

Comment:
Managed to figure this out or can this statement be removed?

Reply:
statement can be removed, hasn't occurred anymore